src/main/java/org/osiam/ResourceServer.java

```diff
@@ -67,13 +55,16 @@
     @Value("${org.osiam.resource-server.db.vendor}")
     private String databaseVendor;
 
-    @Bean
-    public static PropertySourcesPlaceholderConfigurer properties() {
-        PropertySourcesPlaceholderConfigurer propertySources = new PropertySourcesPlaceholderConfigurer();
-        Resource[] resources = new ClassPathResource[] { new ClassPathResource("resource-server.properties") };
-        propertySources.setLocations(resources);
-        propertySources.setIgnoreUnresolvablePlaceholders(true);
-        return propertySources;
+    public static void main(String[] args) {
+        System.setProperty("spring.jpa.hibernate.naming_strategy", "org.hibernate.cfg.ImprovedNamingStrategy");
```

Comment:
Alright, I've found the trouble maker: https://github.com/osiam/resource-server/blob/3b0c5c7142e7ce0b320bb26bf237fc5d9dc016d5/src/main/java/org/osiam/storage/entities/ExtensionFieldValueEntity.java#L45.